x-pack/plugins/data_usage/public/app/hooks/use_charts_url_params.test.tsx

```diff
@@ -5,12 +5,10 @@
  * 2.0.
  */
 
-import moment from 'moment';
-import { METRIC_TYPE_VALUES, MetricTypes } from '../../../common/rest_types';
+import { METRIC_TYPE_VALUES, type MetricTypes } from '../../../common';
 import { getDataUsageMetricsFiltersFromUrlParams } from './use_charts_url_params';
 
-// FLAKY: https://github.com/elastic/kibana/issues/200888
-describe.skip('#getDataUsageMetricsFiltersFromUrlParams', () => {
```

Comment:
@neptunian this was flaky on your PR as well if I recall. I think it should be stable now.